processor/processor.go

```diff
@@ -1289,10 +1370,10 @@ func (proc *Handle) processJobsForDest(partition string, subJobs subJob, parsedE
 	inCountMap := make(map[string]int64)
 	inCountMetadataMap := make(map[string]MetricMetadata)
 	connectionDetailsMap := make(map[string]*types.ConnectionDetails)
-	statusDetailsMap := make(map[string]*types.StatusDetail)
+	statusDetailsMap := make(map[string]map[string]*types.StatusDetail)
```

Comment:
What kind of key does this second map level holds? I find it hard to understand

Reply:
Yeah, complexity is ever growing. 
I think we should plan for complete refactor of reporting metrics